src/view/shell/bottom-bar/BottomBarWeb.tsx

```diff
@@ -240,18 +240,28 @@ const NavItem: React.FC<{
     }
     return getCurrentRoute(state)
   })
+
+  // Détermine si nous sommes sur le profil de quelqu'un d'autre
```

Comment:
Just a head's up: comments in Bluesky's code are in French.

```suggestion
  // Checks whether we're on someone else's profile
```

Reply:
mb thanks !